### ISSUE TITLE
Support for task at startup

### ISSUE THREAD

**Issue by lorenzwalthert** [jwijffels/taskscheduleR#66]
Hi

Thanks for this useful package. I have the following problem: Sometimes, it seems as Windows restarts unexpectedly to install software updates. This might be ok, but the problem is that it leaves running taskscheduleR scrips in a broken state. I'd like to schedule a script at startup to basically restore the state. Hence, I'd need support for "at startup", in addition to the existing methods like "on idle" and "on logon". Also, do you know what happens if I restart and don't log a user in? Does this already trigger "on logon" or does "on logon" mean if I enter my credentials and literally log in. Thanks.

Here is a link with the syntax: https://support.microsoft.com/de-de/help/814596/how-to-use-schtasks-exe-to-schedule-tasks-in-windows-server-2003

**Comment by jwijffels** [comment]
Yes that would be interesting, feel free to contribute with a pull request

**Comment by lorenzwalthert** [comment]
I realized it is more complicated because of permissions needed to execute this (and other problems I can't remember), so we solved the problem otherwise and I won't do a PR.

**Comment by sumedhkumarprasad** [comment]
library(taskscheduleR)
library(shiny)
library(miniUI)
myscript<- "C:/Users/A722646/Desktop/R alert folder/Alert_Code.R"
# myscript <-system.file("extdata", 
#                        "Alert_Code.R",
#                        package = "taskscheduleR")
# ## run script once within 62 seconds
taskscheduler_create(taskname = "myfancyscript_5min", rscript = myscript,
                     schedule = "MINUTE", 
                     starttime = format(Sys.time(),"%H:%M"),
                     modifier = 1)
@lorenzwalthert 
![image](https://user-images.githubusercontent.com/29379271/73446528-f432c980-4382-11ea-94ca-66af37fb9af4.png)
